src/module/item/weapon/values.ts

```diff
@@ -1,22 +1,25 @@
 const WEAPON_CATEGORIES = new Set(["unarmed", "simple", "martial", "advanced"] as const);
 
+const MELEE_OR_RANGED_GROUPS = new Set(["dart", "knife"] as const);
```

Comment:
There's pure ranged knives? Was wondering what this separate property was for and why it can't be in melee weapon groups.

Reply:
Chakram

Reply:
And it is in melee groups

Reply:
Its more of I don't know why there's a separate set if its not used anywhere, but perhaps its for readability. Sure.